Hanan/style.css

```diff
@@ -0,0 +1,105 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+nav{
+    position: relative;
+    background-color:whitesmoke;
+    width: 100%;
+    box-shadow: 0px 0px 30px  black;
+}
+.logo{
+   width: 35px;
+   height: 40px;
+   border-radius: 50%;
+   margin-left: 10px;
+   margin-top: 10px;
+   
+}
+.btn{
+color: rgba(0, 110, 255, 0.884);
+border: 1px solid rgba(0, 110, 255, 0.884);
+border-radius: 12px;
+padding:8px 18px 8px 18px ;
+float: right;
+margin: 10px;
+
+}
+.blue{
+color: white;
+background-color: rgba(0, 110, 255, 0.884);
+}
+main{
+    margin: 0 auto;
+    width: 60%;
+}
+.background{
+    width: 100%;
+    height: 400px;
+    background-repeat: no-repeat;
+    background-size:cover;
+    background-position: center center;
+    position: relative;
+}
+.profile{
+    display: inline-block;
+    position: absolute;
+    width: 120px;
+    height:100px ;
+    border: 2px solid black;
+    border-radius: 50%;
+    margin-top: -50px;
+    margin-left:15px ;
+}
+.name{
+    position: absolute;
```

Comment:
I don't really think you should of used absolute position here, and if you used it you should of actually use (left, right, top, bottom) properties with it